apps/ensadmin/package.json

```diff
@@ -58,11 +57,12 @@
     "graphiql": "^3.8.3",
     "graphql": "^16.10.0",
     "lucide-react": "^0.476.0",
-    "next": "15.4.7",
+    "next": "16.0.0",
     "next-themes": "^0.4.6",
-    "react": "^18",
-    "react-dom": "^18",
+    "react": "19.2.0",
+    "react-dom": "19.2.0",
     "rooks": "^8.4.0",
+    "serve": "^14.2.5",
```

Comment:
Needed to run on Vercel and simulating local production since `next start` runs a server, we're now full SPA.